semantic-conventions/src/opentelemetry/semconv/version.py

```diff
@@ -12,4 +12,4 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
-__version__ = "0.4.1"
+__version__ = "0.5.0"
```

Comment:
i guess the [changelog](https://github.com/open-telemetry/build-tools/blob/main/semantic-conventions/CHANGELOG.md) should also be updated?

Reply:
Oh, I did not realize we had one, yes it should.

Reply:
4720769